lrrbot/commands/quote.py

```diff
@@ -39,13 +39,14 @@ def format_quote(tag, qid, quote, name, date, context):
 		quote_msg += " [{date!s}]".format(date=date)
 	return quote_msg
 
-@bot.command("quote(?: (?:(\d+)|(.+)))?")
+@bot.command("quote\s*(?:(game|show)\s+(.+)|(?:(\d+)|(.+)))?")
 @lrrbot.decorators.sub_only
 @lrrbot.decorators.throttle(60, count=2)
-def quote(lrrbot, conn, event, respond_to, qid, attrib):
+def quote(lrrbot, conn, event, respond_to, meta_param, meta_value, qid, attrib):
 	"""
 	Command: !quote
 	Command: !quote ATTRIB
+	Command: !quote game|show VALUE
 	Section: quotes
 
 	Post a randomly selected quotation, optionally filtered by attribution.
```

Comment:
The command description isn't changed. It should explain what `!quote game|show VALUE` does.